src/ParameterDistributions.jl

```diff
@@ -667,4 +694,145 @@ function transform_unconstrained_to_constrained(
+            if (μ_c - 0.7 * σ_c <= lower_bound)
+                throw(DomainError("`$(name)`: Target std $(σ_c) puts μ - σ too close to lower bound $(lower_bound)"))
+            end
+            if (μ_c + 0.7 * σ_c >= upper_bound)
+                throw(DomainError("`$(name)`: Target std $(σ_c) puts μ + σ too close to upper bound $(upper_bound)"))
+            end
+            μ_u, σ_u = _bounded_gaussian(name, μ_c, σ_c, lower_bound, upper_bound)
+        end
+    end
+    cons = bounded(lower_bound, upper_bound)
+    return ParameterDistribution(Parameterized(Normal(μ_u, σ_u)), cons, name)
+end
+
+function _bounded_gaussian_guess(μ_c::FT, σ_c::FT, lower_bound::FT, upper_bound::FT) where {FT <: Real}
+    # Initial guess for μ_c, σ_c in _bounded_gaussian optimization.
+    # Equations result from inverting the system 
+    # μ_c = f^{-1}(μ_u / sqrt(1 + σ_u^2/2))
+    # σ_c = f^{-1}((μ_u + σ_u/2) / sqrt(1 + σ_u^2/2)) - f^{-1}((μ_u - σ_u/2) / sqrt(1 + σ_u^2/2))
+    # where f^{-1} is the specific form of cons.unconstrained_to_constrained().
+
+    cons = bounded(lower_bound, upper_bound)
+    _Δul = upper_bound - lower_bound
+    _Δum = upper_bound - μ_c
+    _Δml = μ_c - lower_bound
+    init_σ_u =
+        0.5 * (
+            -σ_c * (_Δum^2 + _Δml^2) / (_Δum * _Δml * (σ_c - 1.0)) +
+            hypot(2 * _Δum * _Δml, _Δul * σ_c * (_Δum - _Δml)) / (_Δum * _Δml * abs(σ_c - 1.0))
+        )
+    init_σ_u = 2.0 * log(init_σ_u) / sqrt(1.0 - 2 * log(init_σ_u)^2)
+    init_μ_u = sqrt(1.0 + init_σ_u^2 / 2.0) * cons.constrained_to_unconstrained(μ_c)
+    return (init_μ_u, init_σ_u)
+end
+
+function _bounded_gaussian(name::AbstractString, μ_c::FT, σ_c::FT, lower_bound::FT, upper_bound::FT) where {FT <: Real}
+    # Numerically estimate μ_u, σ_u for unbounded distribution which reproduce desired
+    # μ_c, σ_c in constrained, transformed coordinates. Unlike other cases, can't be solved
+    # for analytically.
+
+    cons = bounded(lower_bound, upper_bound)
+    init_μ_u, init_σ_u = _bounded_gaussian_guess(μ_c, σ_c, lower_bound, upper_bound)
+    # optimize in log σ to avoid constraint σ>0
+    init_logσ_u = log(init_σ_u)
+
+    # Optimize parameters; could speed up with gradient info but don't bother to.
+    # Optimization is finicky since problem becomes singular for large |μ_u|, σ_u: the 
+    # distribution becomes collapsed against the bounds of the interval.
+    function _optim_fn(μlogσ::Vector{Float64})
+        m, s = _mean_std(μlogσ[1], exp(μlogσ[2]), cons)
+        return (m - μ_c)^2 + (s - σ_c)^2
+    end
+    opt = optimize(_optim_fn, [init_μ_u, init_logσ_u], NelderMead(), Optim.Options(x_tol = 1e-5, f_tol = 1e-5))
+    μ_u = Optim.minimizer(opt)[1]
+    σ_u = exp(Optim.minimizer(opt)[2])
+
+    m_c, s_c = _mean_std(μ_u, σ_u, cons)
+    if ~isapprox(μ_c, m_c, atol = 1e-3, rtol = 1e-2)
+        @warn "Unable to set constrained mean for `$(name)`: target = $(μ_c), got $(m_c)"
+    end
+    if ~isapprox(σ_c, s_c, atol = 1e-3, rtol = 1e-2)
```

Comment:
If these are taken in the constrained space, perhaps they should be normalized? (or is that what rtol does?)

Reply:
Sorry, I'm lost on the proper context for "normalized" -- normalization, svd. etc. are only done in CES, right? 

`m_c` and `s_c` are the mean/stddev of the distribution at its final, hopefully optimized parameter values, computed as 1d numerical integrals in the physical, constrained space. The highlighted lines are a sanity check to verify that the optimization set those parameters to values that satisfy the user's requested `μ_c`, `σ_c` for the prior, to very loose tolerances. `rtol` is just the tolerance on the relative difference between the two values.